src/components/DotNavigation/DotNavigation.tsx

```diff
@@ -220,7 +220,7 @@ export class DotNavigation extends React.Component<
       onChange={onChange}
       onKeyDown={onKeyDown}
       aria-label={ariaLabel}
-      {...{ className: styles.extendedRadioButton }}
+      {...{ className: classes.extendedRadioButton }}
```

Comment:
Not related to your work, but i'm really scratching my head on why this was written this way 🤔